qtawesome/styles.py

```diff
@@ -0,0 +1,122 @@
+"""
+QT MODERN
+
+MIT License
+
+Copyright (c) 2017 Gerard Marull-Paretas <gerardmarull@gmail.com>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+from qtpy.QtGui import QPalette, QColor
+from qtpy.QtCore import Qt
+
+
+def dark(app):
+    """ Apply Dark Theme to the Qt application instance.
+
+        Args:
+            app (QApplication): QApplication instance.
+    """
+
+    darkPalette = QPalette()
```

Comment:
```suggestion
    dark_palette = QPalette()
```

Don't use camel case for variable names. Please make the appropriate changes below.